README.md

```diff
@@ -396,12 +396,15 @@ Please look also at [Definitiot of Done](DoD.md) document with further informati
 
 ## Testing
 
-The following tests can be run to test your code in `insights-results-aggregator`. Detailed information about each type of test is included in the corresponding subsection:
+tl;dr: `make before_commit` will run most of the checks by magic
+
+The following tests can be run to test your code in `insights-results-aggregator`.
+Detailed information about each type of test is included in the corresponding subsection:
 
 1. Unit tests: checks behaviour of all units in source code (methods, functions)
-1. REST API Tests: test the real REST API of locally deployed application with database initialized with test data only
-1. Integration tests: the integration tests for `insights-results-aggregator` service
-1. Metrics tests: test whether Prometheus metrics are exposed as expected
+2. REST API Tests: test the real REST API of locally deployed application with database initialized with test data only
```

Comment:
2. etc. is not needed in fact, it is done by Markdown itself

Reply:
didn't know, fixed